tests/ducktape/consumer_strategy.py

```diff
@@ -6,8 +6,35 @@
 """
 import time
 import asyncio
+import json
+import os
 from confluent_kafka import Consumer
 from confluent_kafka.aio import AIOConsumer
+from confluent_kafka.schema_registry import SchemaRegistryClient
+from confluent_kafka.schema_registry._sync.json_schema import JSONDeserializer
+from confluent_kafka.schema_registry._sync.protobuf import ProtobufDeserializer
+from confluent_kafka.schema_registry.avro import AvroDeserializer
+from confluent_kafka.schema_registry._async.json_schema import AsyncJSONDeserializer
+from confluent_kafka.schema_registry._async.protobuf import AsyncProtobufDeserializer
+from confluent_kafka.schema_registry._async.avro import AsyncAvroDeserializer
+from confluent_kafka.schema_registry import AsyncSchemaRegistryClient
+from confluent_kafka.serialization import StringDeserializer, SerializationContext, MessageField
+from tests.integration.schema_registry.data.proto import PublicTestProto_pb2
+
+
+class DeserializedMessage:
+    """Wrapper for messages with deserialized key/value"""
+    def __init__(self, original_msg, key, value):
+        self._original_msg = original_msg
+        self._key = key
+        self._value = value
+
+    def key(self): return self._key
```

Comment:
This would require changing upstream original_msg as well but these should really just be properties instead of methods. If it's like this all the way up the chain we should make an issue for it and fix it later.

Reply:
Yes this setup is not ideal. We would have to update libradkafka API and C-extension to make them properties

Reply:
Can you make a ticket for following up on this, maybe pointing to the code involved?